lib/sles4sap_publiccloud.pm

```diff
@@ -831,4 +836,98 @@ sub list_cluster_nodes {
     die 'None of the hosts are currently part of existing cluster' unless @cluster_nodes;
 }
 
+=head2 get_hana_database_status
+
+    Parses "hdbsql" command output.
+    Returns 1 if hana database is online, 0 otherwise.
+
+=back
```

Comment:
you have a `=back` that is not needed

`=back` is needed when you have a previous `=over`. You do not have an `=over` so you can remove the `=back`

here an example using back and over

```
=head3 qesap_az_calculate_address_range

Calculate the vnet and subnet address
ranges. The format is 10.ip2.ip3.0/21 and
 /24 respectively. ip2 and ip3 are calculated
 using the slot number as seed.

=over 1

=item B<SLOT> - integer to be used as seed in calculating addresses

=back

=cut
```

Reply:
thx